keyboards/pk60/pk60.c

```diff
@@ -1 +1,11 @@
 #include "pk60.h"
+
+void led_set_user(uint8_t usb_led) {
```

Comment:
My recommendation is to actually set this in the keymap.c file (the default keymap), so that people can customize it, if they wish.

Otherwise, you would want to use this here, instead:
```c
void led_set_kb(uint8_t usb_led) {
    
    if (usb_led & (1<<USB_LED_CAPS_LOCK)) {
        PORTF |= (1<<4);
    } else {
        PORTF &= ~(1<<4);
    }
    led_set_user(usb_led);
}
```

This way, it's set at the keyboard level with the option to customize at the user level still.

Reply:
Thanks!